src/components/Admin/managementData.ts

```diff
@@ -117,6 +117,15 @@ export const managementData = {
         { description: 'Updates', url: '/community/updates' },
       ],
     },
+    {
+      name: 'Authorization',
+      buttons: [
+        {
+          description: 'Opportunity admins',
+          url: '/authorization/admins',
```

Comment:
if we are going to stop using the credential in the URL, isn't it better to just be `/admins` or `/authorization`?

Reply:
Keep it for now. Will be useful very soon.